### ISSUE TITLE
As an expert integrator, drill down into technical logs for integrations

### ISSUE THREAD

**Issue by kcbabo** [syndesisio/syndesis#1094]
As an expert integrator, I want the ability to drill down into technical log information that's reported as part of the integration runtime container.  Ideally, the integration details page should provide a link which takes me directly to the pod log view for a specific integration.

**Comment by chirino** [comment]
Did a little testing to see if we can deep link into the openshift consoles and it seem to mostly work.

Scenario prep:
Log into openshift console and and go to a pod's log screen and copy the URL.

Scenario 1:
In an incognito console, log into syndesis console.  In new tab paste pod log URL.  You end up getting an openshift log on screen.  Logging in again, then takes you the pod's logs.  You can't seem to link to specific pod log lines.

Scenario 2: 
While in the same incognito session, redeploy the integration.  This kills the previous pod.  Use the previous pod log URL in a new tab.  You get this:

![banners_and_alerts_and_openshift-jvm](https://user-images.githubusercontent.com/103255/35048033-dc91ab88-fb69-11e7-9755-536aa1b50f84.jpg)

So basically we should only link to pods that are still running.
